### PR TITLE
chore(tests): disable parseSQL in http logictest

### LINKED ISSUE #7027: `logictest` will mistake converting double quote `"` to single quote `'` in query when testing with http protocol
https://github.com/datafuselabs/databend/blob/43cbd00efd425c1c2eb42a1f3815282754b28873/tests/logictest/http_connector.py#L121-L129

cc @ZeaLoVe

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

1. disable `parseSQL` in http logictest,  Fixes #7027
2. merge some unused DDL test case to one file

### CODE REVIEW

**Comment by ZeaLoVe** [comment]
Can clickhouse connector disable parseSQL？

**Review by Xuanwo — APPROVED** [review]

**Review by BohuTANG — APPROVED** [review]

**Comment by BohuTANG** [comment]
```
Parsed Statement
    at_line: 99,
    s_type: Statement: ok, type: None,
    suite_name: base/03_dml/03_0018_insert_into_variant,
    text:
        INSERT INTO t3 SELECT 1, parse_json('{"a":1,"b":{"k":2},"c":[10,11,12]}');
    results: [],
    runs_on: {'clickhouse', 'mysql', 'http'},
---------------------------------------------
Runner: clickhouse
ErrorType: statement ok execute with exception
```

**Comment by sundy-li** [comment]
using the `sqlalchemy` interface
```
SELECT parse_json('{"a":1,"b":2}')::object;
```
will be compiled into

```
SELECT parse_json('{"a"%(1)s,"b"%(2)s}')::object;
```

File: tests/logictest/suites/base/02_function/02_0051_function_semi_structureds_get [hunk 1]
```diff
@@ -11,13 +11,13 @@ select get(parse_json('[2.71, 3.14]'), 2);
...
 statement query T
-select get(parse_json('{"aa":1, "aA":2, "Aa":3}'), 'aA');
+select get(parse_json('{"aa": 1, "aA":2, "Aa":3}'), 'aA');
 
 ----
 2
 
 statement query T
-select get(parse_json('{"aa":1, "aA":2, "Aa":3}'), 'AA');
+select get(parse_json('{"aa": 1, "aA":2, "Aa":3}'), 'AA');
```
Comment (ZeaLoVe) [inline tests/logictest/suites/base/02_function/02_0051_function_semi_structureds_get:20]: Is aA , Aa also needs empty space？

File: tests/logictest/suites/base/02_function/02_0051_function_semi_structureds_get [hunk 2]
```diff
@@ -11,13 +11,13 @@ select get(parse_json('[2.71, 3.14]'), 2);
 NULL
 
 statement query T
-select get(parse_json('{"aa":1, "aA":2, "Aa":3}'), 'aA');
+select get(parse_json('{"aa": 1, "aA":2, "Aa":3}'), 'aA');
```
Comment (ZeaLoVe) [inline tests/logictest/suites/base/02_function/02_0051_function_semi_structureds_get:14]: ditto

**Pull request merged**